src/server/plugins/engine/pageControllers/SummaryPageController.ts

```diff
@@ -158,12 +158,10 @@ export class SummaryPageController extends PageController {
 
       viewModel.backLink = this.getBackLink(progress)
 
-      const missingEmailWarning =
+      const notificationEmailWarning =
         await this.buildMissingEmailWarningModel(request)
 
-      if (missingEmailWarning) {
-        return h.view('summary', { ...viewModel, ...missingEmailWarning })
-      }
+      viewModel.notificationEmailWarning = notificationEmailWarning
```

Comment:
`notificationEmailWarning` isn't even neccessary as a variable any more, you could directly assign the result to `viewModel.notificationEmailWarning`.